### PR TITLE
Panel fixes

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

@jbhoosreddy and I looked at some panel bugs today and fixed some of the big ones

* gap in the welcome box
* jump to line when selecting a breakpoint
* jump to line when stepping
* gutter disappearing while stepping

I'll add some mochitests for the gutter and jumping, they're tougher to test because we need to actually test to see if the line is "visible" not just highlighted 

screen shot 2017-11-05 at 6 12 58 pm

### CODE REVIEW

**Comment by jasonLaster** [comment]
this is getting closer, working on some mochitests to prevent future regressions

File: src/actions/sources.js [hunk 1]
```diff
@@ -322,28 +322,23 @@ export function togglePrettyPrint(sourceId: string) {
...
 
     const selectedLocation = getSelectedLocation(getState());
-    const selectedOriginalLocation = selectedLocation
-      ? await sourceMaps.getOriginalLocation(selectedLocation)
-      : {};
-
     const url = getPrettySourceURL(source.url);
     const prettySource = getSourceByURL(getState(), url);
 
+    let options = {};
```
Comment (codehag) [inline src/actions/sources.js]: this can be `const` (im surprised linter isnt complaining!)

File: src/components/Editor/Editor.css [hunk 2]
```diff
@@ -30,9 +30,9 @@
  */
 .editor-wrapper {
   position: absolute;
-  height: calc(100% - 29px);
+  height: calc(100% - 30px);
```
Comment (codehag) [inline src/components/Editor/Editor.css]: this seems like a magic number, maybe lets make this a var?

File: src/test/mochitest/head.js [hunk 3]
```diff
@@ -784,10 +788,54 @@ function type(dbg, string) {
...
+ * Checks to see if the inner element is visible inside the editor.
+ *
+ * @memberof mochitest/helpers
+ * @param {Object} dbg
+ * @param {HTMLElement} inner element
+ * @return {boolean}
+ * @static
+ */
+
+function isVisibleInEditor(dbg, element) {
```
Comment (codehag) [inline src/test/mochitest/head.js]: :+1:

File: src/components/Editor/Editor.css (hunk 2 above)
Comment (jasonLaster) [reply src/components/Editor/Editor.css]: cool

**Pull request merged**